packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswerCopyToClipboard/quanticGeneratedAnswerCopyToClipboard.js

```diff
@@ -5,7 +5,7 @@ import {LightningElement, api} from 'lwc';
 
 /**
  * The `QuanticGeneratedAnswerCopyToClipboard` component displays a button responsible for copying the generated answer to clipboard.
- * @category Internal
+ * @category Search
```

Comment:
I think I would keep this component internal cause the main purpose behind creating it separately was to reduce the complexity  of the main component, the `QuanticGeneratedAnswer` component. 
Thus I think we shouldn't overload our documentation with a component that is unlikely to be handled directly by the end users.

What do you think @lbergeron  @erocheleau  @SimonMilord ?